.gitignore

```diff
@@ -4,3 +4,6 @@ dist/
 coverage/
 .config/
 .npm/
+
+# docker run bash history
+.ash_history
```

Comment:
Should this be `.bash_history`?

Reply:
No - it's the bash history file from running in a container that gets dropped on the host when running `docker run -it <container> /bin/sh`